### PR TITLE
Added a testing scenario for Event

### PR DESCRIPTION
**Pull request by mk-sgent**

### CODE REVIEW

**Comment by mk-sgent** [comment]
EventPicturefozRazvan

File: sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/api/PersonSteps.java [hunk 1]
```diff
@@ -50,6 +50,7 @@ public PersonSteps(
           Person createPersonObject = personApiService.buildGeneratedPerson();
           apiState.setLastCreatedPerson(createPersonObject);
           personsHelper.createNewPerson(createPersonObject);
+          System.out.println("Year of Birth" + createPersonObject.getBirthdateYYYY().toString());
```
Comment (rdutu-vg) [inline sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/api/PersonSteps.java:53]: remove

File: sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/events/EventDirectorySteps.java [hunk 2]
```diff
@@ -18,31 +18,121 @@
...
+              dataOperations.getPartialUuidFromAssociatedLink(eventUuid));
+        });
+
     When(
         "I click on the NEW EVENT button",
         () ->
             webDriverHelpers.clickWhileOtherButtonIsDisplayed(
                 EventDirectoryPage.NEW_EVENT_BUTTON, TITLE_INPUT));
+    When(
+        "I select Risk level filter {string} among the filter options",
```
Comment (rdutu-vg) [inline sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/events/EventDirectorySteps.java:63]: mention also page llocation

File: sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/events/EventDirectorySteps.java [hunk 3]
```diff
@@ -18,31 +18,121 @@
...
+        (String sourceType) -> {
+          // String sourceType = apiState.getCreatedEvent().getSrcType();
+          webDriverHelpers.waitForPageLoaded();
+          webDriverHelpers.selectFromCombobox(
+              FILTER_BY_SOURCE_TYPE, SourceTypeValues.getCaptionForName((sourceType)));
+        });
+    When(
+        "I select Type of Place field {string} among the filter options",
+        (String sourceTypeOfPlace) -> {
+          // String sourceTypeOfPlace = apiState.getCreatedEvent().getTypeOfPlace();
```
Comment (rdutu-vg) [inline sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/events/EventDirectorySteps.java:131]: remove commented and add location in bdd name

File: sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/persons/PersonDirectorySteps.java [hunk 4]
```diff
@@ -64,6 +69,33 @@ public PersonDirectorySteps(
...
+
+    Then(
+        "I check that number of displayed Persons results is {int}",
+        (Integer number) ->
+            assertHelpers.assertWithPoll20Second(
+                () ->
+                    Assert.assertEquals(
+                        webDriverHelpers.getNumberOfElements(CASE_GRID_RESULTS_ROWS),
+                        number.intValue(),
+                        "Number of displayed cases is not correct")));
```
Comment (rdutu-vg) [inline sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/persons/PersonDirectorySteps.java:97]: persons not cases

**Review by rdutu-vg — CHANGES_REQUESTED** [review]

**Review by rdutu-vg — APPROVED** [review]

**Pull request merged**